packages/core/src/families/celo/validation.ts

```diff
@@ -0,0 +1,24 @@
+import { z } from "zod";
+import { schemaFamilies, schemaTransactionCommon } from "../common";
+
+export const OPERATION_MODE = [
+  "send",
+  "lock",
+  "unlock",
+  "withdraw",
+  "vote",
+  "revoke",
+  "activate",
+  "register",
+] as const;
```

Comment:
Why not `z.enum()` instead?

Tron family has something similar.

https://github.com/LedgerHQ/wallet-api/blob/main/packages/core/src/families/tron/validation.ts#LL4..LL10